addons/main.go

```diff
@@ -147,6 +148,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	//name of all the validating webhook configurations. the following need to be populatec with the names of the
+	//validating and webhooks configurations.
+	var vWhCfgNames []string
+	var mWhCfgNames []string
+	webhook.SetupWebhooks(&mgr, addonNamespace, vWhCfgNames, mWhCfgNames)
```

Comment:
Question: Do we want to handle the error here? Or we want to proceed when it fails to set up webhook.

Reply:
Yes. we probably should. But the question is what we should do? 
Is not catastrophic failure. Addon manager can still do its job. The user won't be able to use the api to configure the packages.